client/dashboard/sites/add-new-site/index.tsx

```diff
@@ -24,6 +27,9 @@ import './style.scss';
 function AddNewSite( { context }: AddNewSiteProps ) {
 	const { recordTracksEvent } = useAnalytics();
 	const { onboardingLinks } = useAppContext();
+	const auth = useContext( AuthContext );
+	const user = auth?.user;
```

Comment:
Drive by comment while this is still in draft: I think this can be replaced with

```suggestion
	const { user } = useAuth();
```

We probably shouldn't even have `AuthContext` exported tbh. I bet it is there for some unit test.

Reply:
Thanks for for the tip.
I refactored to use the `useAuth` and `AuthProvider`.